### PR TITLE
Add fr.openfisca.org/api/v18 to serve the official url  API

### PR DESCRIPTION
**Pull request by Anna-Livia**
Connected to openfisca/openfisca-core#570
This PR changes the server configuration to serve the new API on fr.openfisca.org/api/v18.
API version # should be increase at each Openfisca-France major version bump.

### CODE REVIEW

File: fr.openfisca.org/fr.openfisca.org.conf [hunk 1]
```diff
@@ -28,6 +28,13 @@ server {
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
```
Comment (fpagnoux) [inline fr.openfisca.org/fr.openfisca.org.conf]: Could we add a comment on this block to indicate it only stays here for retro-compatibility ?

Comment (Anna-Livia) [reply fr.openfisca.org/fr.openfisca.org.conf]: Being curious : retro-compatibility with what ?

Comment (fpagnoux) [reply fr.openfisca.org/fr.openfisca.org.conf]: Retro-compatibility for people who may be using `https://fr.openfisca.org/api/v2`

Comment (fpagnoux) [reply fr.openfisca.org/fr.openfisca.org.conf]: If we don't want retro-compatibility then this block should be removed (once the leg explorer is merged).

**Pull request merged**